WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1478,6 +1478,23 @@ ContactListPaginationChild          = display:inline-block;
 ContactListPaginationLink           = 
 ContactListPaginationLinkIcon       = height:30px; width:30px;
 
+# Contact Search
+
+ContactSearchListContainerHeight       = 300px;
```

Comment:
My idea of having it was in case if it can be reused.
Removed, now.
https://github.com/Zimbra/zm-web-client/pull/117/commits/97cfa50df7bb0d1b69e5dbf78f80541dc101f0d3